bootstrap-tests.js

```diff
@@ -28,9 +41,11 @@ function writeFixtures() {
 	for (let key of Object.keys(fixtures)) {
 		const jsx = fixtures[key].jsx;
 		const wrapper = render(jsx);
-		const jsxCode = `(${markupToOneLine(reactElementToJsx(jsx))})`;
+		let jsxCode = `(${markupToOneLine(reactElementToJsx(jsx))})`;
 		const htmlCode = JSON.stringify(wrapper.html());
-
+		for (let dataKey of Object.keys(recognizedDataStrings)) {
+			jsxCode = replaceAll(recognizedDataStrings[dataKey], dataKey, jsxCode);
```

Comment:
@borisyankov This allows us to write `data={sampleData}` in `fixtures.js` rather than `data={[0.26789611283279424, ...]}` every time. Yes, I used string replacement to do this. It's slow and kind of inelegant but it works :) Which to taste is good enough for this kind of dev script.